vsts.yml

```diff
@@ -27,7 +27,20 @@ phases:
 
   steps:
   - bash: |
-     echo "##vso[task.setvariable variable=gitcommit]$BUILD_SOURCEVERSION"
+     if [[ -z "${SYSTEM_PULLREQUEST_PULLREQUESTNUMBER}" ]] || "$SYSTEM_PULLREQUEST_ISFORK" == "True"; then
```

Comment:
Does the second clause here also need test brackets? e.g. 

`... || [ "${SYSTEM_PULLREQUEST_ISFORK}" == "True" ]; then`

Reply:
They technically aren't needed for either one, though I guess we should be consistent.